modules/openapi-generator/src/test/resources/3_0/csharp/hello-world.yaml

```diff
@@ -0,0 +1,28 @@
+openapi: 3.1.0
```

Comment:
shall we put this under `modules/openapi-generator/src/test/resources/3_1/` which stores all 3.1 spec at the moment?

Reply:
done